src/main/resources/org/jenkinsci/plugins/github_branch_source/GitHubSCMSource/help-buildForkPRHead.html

```diff
@@ -0,0 +1,4 @@
+<div>
+    Whether to build pull requests filed from forks of the main repository.
```

Comment:
🐜 I recommend you to use `<p>` in order to avoid [this|https://github.com/jenkinsci/github-branch-source-plugin/pull/60#issuecomment-227539086]. The information appears tight.